lib/mongodb-filter-wrapper.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+class MongoDBFilterWrapper {
+	static get filtersAllowed() {
+		return {
+			equal: '$eq',
+			not: '$ne',
+			greater: '$gt',
+			greaterOrEqual: '$gte',
+			lesser: '$lt',
+			lesserOrEqual: '$lte',
+			in: '$in',
+			notIn: '$nin',
+			reg: '$regex',
+			all: '$all'
+		};
+	}
+
+	/**
+		* Enter door to filter process
+		* @param model {Object} Model that will use the filter
+		* @param filters {Object} Filters defined as parameters
+		* @returns {Object} with the filter parsed
+		*/
+	static process(model, filters) {
+
+		// Validate filters given
+		if(typeof filters !== 'object' || (Array.isArray(filters) && filters.length === 0) || Object.entries(filters).length === 0)
```

Comment:
```suggestion
		if(typeof filters !== 'object' || (Array.isArray(filters) && !filters.length) || !Object.entries(filters).length)
```
No es necesario utilizar `length === 0` simplemente con `!length` estas haciendo lo mismo para este caso